### DISCUSSION TITLE
How to query for uuid in an array?

### DISCUSSION THREAD

**Discussion by jesper-bylund** [supabase/supabase#3473] [Questions] [answered]
Hi,
I have a datatype that looks something like this

```js
const model = {
  id: uuid,
  reference: uuid[]
}
```

But I cannot figure out a way to query for this using the normal syntax.  🤔 I there a way?

My current code:
```js
supabaseClient
  .from("notes")
  .select("*")
  .in('references', ["583104d5-xxxxxx-131a40531afc"]);
```

returns the quite logical error: 
"could not find array type for data type uuid[]"

**Comment by jesper-bylund** [comment]
I also can't seem to create a function for this.. 🤔 Seems like the syntax is a bit obscure and hard to google.

<img width="653" alt="Screenshot 2021-10-10 at 13 01 42" src="https://user-images.githubusercontent.com/186083/136692861-974e695c-61a0-4013-9737-1ec54a9bee32.png">
<img width="947" alt="Screenshot 2021-10-10 at 13 02 53" src="https://user-images.githubusercontent.com/186083/136692864-32232cae-6bfa-4ade-8f73-c5af998647e0.png">

**Reply by GaryAustin1** [reply]
Try .contains.  
 .in means search column for anything in this array of values.

**Reply by jesper-bylund** [reply]
@GaryAustin1 amazing! That seems to have worked perfectly! Thank you Add it as an Answer and I'll mark this as solved.

**Comment by GaryAustin1** [comment, marked as answer]
Still new to Github.... adding so can be answer....
Try .contains.
.in means search column for anything in this array of values.

**Reply by jesper-bylund** [reply]
Thanks Gary!
